### PR TITLE
New Topic Outline on Preparing Local Project to Be Production Ready Issue #2107

### PR DESCRIPTION
**Pull request by jsnorek**
This is a PR for issue #2107: The goal for this issue was to create a topic outline on preparing local projects to be production ready. 

I created a new folder for housing the necessary coding screenshots for the production environment walkthrough as well as the file for the topic outline.

Screenshot 2024-12-13 at 3 35 59 PM

In this new file I outlined **prerequisites** pulled from the other deployment files as they felt appropriate here. I also added **learning objectives**, basic information on **local environment vs production environment vs deployment**, and the **"Build Your Production Instructions"** from the specified [doc](https://docs.google.com/document/d/1ley6fg1x44w5Env4a8CKOpaPQNqCPkr8wagtiGOhEHI/edit?tab=t.0).

Screenshot 2024-12-13 at 3 43 37 PM

The other [linked doc](https://docs.google.com/document/d/1JIhYrbk5LPjGQzxcGRBwvVpGztoh4fgUW4ITHGHqEzs/edit?tab=t.0) from the issue had mostly the same information as the production instructions doc.

I feel as though the production instructions are a little difficult to follow/understand without the reader working on the exact project referenced, but I am not sure which project that is so I can link it.

### CODE REVIEW

File: deploying/deploying-and-production-ready.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+**Local Environment**: This is a coding workspace on the developer's computer. Since it is connected to the developer's local drive, this environment is meant for development and testing, not for deployment. Instead, a production environment is needed for live deployment.
+
+**Production Environment**: This is the environment where your application is fully live and accessible to users on the web. Setting up the production environment allows you to deploy your code into a real-world setting.
+
+**Deployment**: This is the entire process of moving your code from development to its final destination, production. It can involve steps like copying files, setting up your server, and making the application available to users.
+
+## Build Your Production Instructions
+
+These instructions will help you to have a production environment different from your local dev. This will help you to have a place to show your last working code while having a different space to keep working on improving your project.
```
[the remark this answers is not in the document]
Comment (jsnorek) [reply deploying/deploying-and-production-ready.md:28]: I have added some clarifying text to the instructions and linked the example project template for easy reference in this commit https://github.com/Techtonica/curriculum/pull/2134/commits/af02bf62c4a6c0cc2a5d5f67a1247defb4a59049

[the remark this answers is not in the document]
Comment (daaimah123) [reply deploying/deploying-and-production-ready.md:28]: Great additions! Marking as resolved!

**Review by daaimah123 — APPROVED** [review]

**Comment by jsnorek** [comment]
@marjanasarker @ChasVanDav could you please review when you have time and let me know your thoughts? 👍

**Review by marjanasarker — APPROVED** [review]
lgtm

**Comment by daaimah123** [comment]
@jsnorek 🥳 

1. Please acknowledge and "resolve" any outstanding comments that you have not interacted with.
2. Please let me know if you have any additional incoming changes. If not, go ahead and "Merge pull request".
3. Ensure both your GH issue and PR status has been appropriate updated (this should happen automatically, but if not do so manually).
4. Leave a respective comment on your GH issue closing it.

**Comment by ChasVanDav** [comment]
Great job, Jules!

**Review by ChasVanDav — APPROVED** [review]

**Comment by jsnorek** [comment]
@ChasVanDav @marjanasarker thanks for reviewing!

@daaimah123 there are no other changes. I'm going to merge 👍

**Pull request merged**